src/main/java/it/pagopa/transactions/commands/bean/NewTransactionRequestData.java

```diff
@@ -0,0 +1,15 @@
+package it.pagopa.transactions.commands.bean;
```

Comment:
What's the difference between `it.pagopa.transactions.commands.bean` and `it.pagopa.transactions.commands.data`?

Reply:
Fixed with 815b383d2b052af9a6a0103e440139ec9e4afdc4.
Effectively the package bean was useless.
Thank you 🙂